misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/src/main/java/de/metas/camel/externalsystems/shopware6/order/ImportOrdersRouteContext.java

```diff
@@ -365,4 +372,8 @@ private String getSalutationDisplayNameById(@Nullable final String salutationId)
 
 		return salutationInfoProvider.getDisplayNameBySalutationId(salutationId);
 	}
+
+	public void incrementPageIndex() {
```

Comment:
please make sure to have "the right" formatting rules. (=>https://github.com/metasfresh/metasfresh/tree/master/misc/dev-support)